CODEOWNERS

```diff
@@ -1 +1 @@
-* @hillairet @Nathan-Nesbitt
+* @hillairet @lishanl
```

Comment:
Interesting [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) features. We can also tag the team to be responsible for specific files/folders/branches. 

Would this addition here automatically assign me as a reviewer to the future PRs? Maybe a comment would make that clear.